Assets/MRTK/Core/Utilities/BuildAndDeploy/UwpPlayerBuildTools.cs

```diff
@@ -88,6 +174,18 @@ async void PostBuildAction(IBuildInfo innerBuildInfo, BuildReport buildReport)
                     UwpAppxBuildTools.AddCapabilities(uwpBuildInfo);
                     UwpAppxBuildTools.UpdateAssemblyCSharpProject(uwpBuildInfo);
 
+                    if (!string.IsNullOrEmpty(BuildDeployPreferences.AppLauncherModelLocation))
+                    {
+                        string appxPath = $"{buildDirectory}/{PlayerSettings.productName}";
+
+                        Debug.Log($"3D App Launcher: {BuildDeployPreferences.AppLauncherModelLocation}, Destination: {appxPath}/{AppLauncherPath}");
```

Comment:
Extraneous debug line?

Reply:
I left this one in to make it clear we were copying an additional file to the solution folder, but I'm happy to remove!

Reply:
I think it's fine to keep then, we do the same with defaultLinkXmlContents for the link.xml file in the generated folder.